eoscostarica/guide.eoscostarica.io
# File(language selector): Add languege selector

### Language Selector

#54 

### Steps to test
1. Run local project
1. See website structure details


#### CheckList
